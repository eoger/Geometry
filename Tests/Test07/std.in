C Cercle1 5 -5 +5
C Cercle1 5 -5 +65
LIST
CLEAR
R Rect2 5 -5 +5 -5
R Rect2 5 -5 +5 -5
LIST
CLEAR
L Ligne3 5 -8 +9 -4
L Ligne3 5 -8 +9 -4
LIST
CLEAR
PL poly4 -5 5 +3 2 4 8 +5 -9
PL poly4 -5 5 +3 2 4 8 +5 -9
LIST
CLEAR
C Cercle1 5 -5 +5
L Ligne3 5 -8 +9 -4
OA 5OA Cercle1 Ligne3
OA 5OA Cercle1 Ligne3
LIST
CLEAR
EXIT