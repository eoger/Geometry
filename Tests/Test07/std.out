OK
ERR
C Cercle1 5 -5 5
OK
OK
ERR
R Rect2 5 -5 5 -5
OK
OK
ERR
L Ligne3 5 -8 9 -4
OK
OK
ERR
PL poly4 -5 5 3 2 4 8 5 -9
OK
OK
OK
OK
ERR
OA 5OA Cercle1 Ligne3
C Cercle1 5 -5 5
L Ligne3 5 -8 9 -4
OK
